apps/ios/LiveVideo/LiveVideo/Views/StreamConfigFlow/EnterStreamNameView.swift

```diff
@@ -8,59 +8,75 @@ struct EnterStreamNameView: View {
     private struct ViewModel {
         let title: String
         let tip: String
+        let shouldShowRecordOption: Bool
         let shouldSelectRole: Bool
     }
     
     @EnvironmentObject var flowModel: StreamConfigFlowModel
     @EnvironmentObject var authManager: AuthManager
     @State private var streamName = ""
     @State private var isShowingSelectRole = false
+    @State private var shouldRecord = false
 
     private var viewModel: ViewModel {
         switch flowModel.parameters.role {
         case .host:
             return ViewModel(
                 title: "Create new event",
-                tip: "Tip: give your event a name that’s related to the topic you’ll be talking about.",
+                tip: "Give your event a name that’s related to the topic you’ll be talking about.",
+                shouldShowRecordOption: true,
                 shouldSelectRole: false
             )
         case .none, .viewer, .speaker:
             return ViewModel(
                 title: "Join event",
                 tip: "Enter the event name.",
+                shouldShowRecordOption: false,
                 shouldSelectRole: true
             )
         }
     }
 
     var body: some View {
         NavigationView {
-            FormStack {
-                Text(viewModel.tip)
-                    .modifier(TipStyle())
-
-                TextField("Event name", text: $streamName)
-                    .textFieldStyle(FormTextFieldStyle())
-                    .autocapitalization(.none)
-                    .disableAutocorrection(true)
-
-                VStack {
-                    NavigationLink(destination: SelectRoleView(), isActive: $isShowingSelectRole) {
-                        EmptyView()
+            Form {
```

Comment:
Realized that SwiftUI has a form built in so going to try to use it for input screens like this. It looks quite nice.